.soliumrc.json

```diff
@@ -16,8 +16,8 @@
       "warning",
       5
     ],
-    "error-reason": [
-      "off"
-    ]
+    "error-reason": 0,
+    "security/no-assign-params": 0,
+    "security/no-inline-assembly": 0
```

Comment:
This exceptions, wouldn't make sense to disable in the point where we have it in the code, instead of for the whole project?